.github/workflows/maven.yml

```diff
@@ -9,13 +9,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Set up JDK 9
+    - name: Set up JDK 14
       uses: actions/setup-java@v1
       with:
-        java-version: 9
+        java-version: 14
```

Comment:
Does it mean that compilation level is 14? We should use JDK 14, but compile level should be 8 for now (until we change that requirement) in order to be compatible with old CPXs

Reply:
It's basically just a version of jdk that github is using for the build, but the compilation level is set in pom file - target version is 8. It is not really critical which version to have in here, but previously it's been 9 so I just figured it's better to get it aligned with either 8 or 14. So if you think this one is better to be kept as 8 - I will change it down to 8

Reply:
Understood. Leave it 14 then.